tutor/src/components/annotations/annotation.jsx

```diff
@@ -347,64 +338,52 @@ export default class AnnotationWidget extends React.Component {
   @action.bound
   updateActiveAnnotation(event) {
     const newValue = event.target.value;
-
-    this.activeHighlight.annotation = newValue;
+    this.activeHighlight.text = newValue;
```

Comment:
lint on osweb insists on a blank line after declarations. :)

Reply:
oh that's a nice rule.  We should probably conduct a "lint rules" summit and share our configs to achieve a standard.